diff -ru paramiko-2.6.0/tests/test_client.py paramiko-2.6.0-a/tests/test_client.py
--- paramiko-2.6.0/tests/test_client.py	2019-06-24 06:48:30.000000000 +0800
+++ paramiko-2.6.0-a/tests/test_client.py	2019-07-26 02:51:15.154244423 +0800
@@ -33,7 +33,7 @@
 import weakref
 from tempfile import mkstemp
 
-from pytest_relaxed import raises
+from pytest import raises
 from mock import patch, Mock
 
 import paramiko
@@ -684,10 +684,10 @@
 
     # TODO: more granular exception pending #387; should be signaling "no auth
     # methods available" because no key and no password
-    @raises(SSHException)
     def test_passphrase_kwarg_not_used_for_password_auth(self):
-        # Using the "right" password in the "wrong" field shouldn't work.
-        self._test_connection(passphrase="pygmalion")
+        with raises(SSHException):
+            # Using the "right" password in the "wrong" field shouldn't work.
+            self._test_connection(passphrase='pygmalion')
 
     def test_passphrase_kwarg_used_for_key_passphrase(self):
         # Straightforward again, with new passphrase kwarg.
@@ -705,14 +705,14 @@
             password="television",
         )
 
-    @raises(AuthenticationException)  # TODO: more granular
     def test_password_kwarg_not_used_for_passphrase_when_passphrase_kwarg_given(  # noqa
         self
     ):
-        # Sanity: if we're given both fields, the password field is NOT used as
-        # a passphrase.
-        self._test_connection(
-            key_filename=_support("test_rsa_password.key"),
-            password="television",
-            passphrase="wat? lol no",
-        )
+        with raises(AuthenticationException):  # TODO: more granular
+            # Sanity: if we're given both fields, the password field is NOT used as
+            # a passphrase.
+            self._test_connection(
+                key_filename=_support("test_rsa_password.key"),
+                password="television",
+                passphrase="wat? lol no",
+            )
